recipes/bash-git-prompt/bld.bat

```diff
@@ -0,0 +1,8 @@
+copy "%RECIPE_DIR%\build.sh" .
+set PREFIX=%PREFIX:\=/%
```

Comment:
On windows you want to install  the share directory to `LIBRARY_PREFIX`, but the activate scripts still stay under `PREFIX`.